### PR TITLE
Dok 141479

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Update link to OAN Manager

### Resolves:
 [OKTA-141479](https://oktainc.atlassian.net/browse/OKTA-141479)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Karan, can you review the link to make sure it's right? I tested in my local, but a review is always good. Jordan, if this passes travis, can you review the output files? This is a hotfix for a link that was fixed (or so I thought!) before Oktane.

File: Gemfile.lock [hunk 1]
```diff
@@ -189,6 +189,7 @@ GEM
       gemoji (~> 3.0)
       html-pipeline (~> 2.2)
       jekyll (>= 3.0)
+    json (2.1.0)
```
Comment (jmelberg-okta) [inline Gemfile.lock]: I think your machine is in a weird state here, as these dependencies continue to change. Can you run:
`bundle install` before running `npm run build-prod` to see if this removes the `json` dependencies?

Comment (mystiberry-okta) [reply Gemfile.lock]: Sure. They were gone at one point. I did a git pull on master before creating this branch.

Comment (mystiberry-okta) [reply Gemfile.lock]: Did that and still got json in the Gemfile.lock.

**Comment by mystiberry-okta** [comment]
I rebased and rebuilt, but no changes, so no new production files.

**Pull request merged**